ckanext/validation/jobs.py

```diff
@@ -40,6 +40,11 @@ def run_validation_job(resource):
     options = t.config.get(
         u'ckanext.validation.default_validation_options')
     if options:
+        # Somtimes json.loads(options) fails with No JSON object could be decoded
+        # When that is the case eval(options) returns string, otherwise dict
+        # see https://github.com/frictionlessdata/ckanext-validation/issues/42
+        if isinstance(eval(options), str):
```

Comment:
I wasn't able to understand this issue but I would suggest considering to use `ast.literal_eval` instead of `eval`.

Reply:
Instead of `json.loads()` right? Agree, just wanted to touch the original code as less as possible...

Reply:
instead of `eval`...